src/main/java/com/xceptance/xlt/engine/socket/XltSockets.java

```diff
@@ -46,9 +51,9 @@ public final class XltSockets
                 // set the global socket impl factory
                 Socket.setSocketImplFactory(new InstrumentedSocketImplFactory());
             }
-            catch (final IOException ex)
+            catch (final Throwable ex)
             {
-                throw new RuntimeException("Failed to initialize XLT sockets", ex);
+                LOG.warn("Failed to initialize XLT sockets: {}", ExceptionUtils.getRootCauseMessage(ex));
```

Comment:
Logging a warning might be fine for development mode but when running in context of a load test, we may log it as error (or even re-throw it) instead. WDYT? Otherwise, load test environment misconfigurations might be easily overseen.

Reply:
Nice. Thx.